lib/apolloClient.ts

```diff
@@ -0,0 +1,28 @@
+import { ApolloClient, createHttpLink, InMemoryCache } from '@apollo/client';
+import { setContext } from '@apollo/client/link/context';
+
+const httpLink = createHttpLink({
+  uri: process.env.uri,
+});
+
+const authLink = setContext((_, { headers }) => {
+  console.log(process.env.uri);
```

Comment:
You want to remove the `console.log`, or add something like the [debug](https://www.npmjs.com/package/debug) library to print only in debug mode